src/site/_includes/components/PostCard.js

```diff
@@ -119,29 +125,48 @@ module.exports = ({post}) => {
     `;
   }
 
+  function renderChips() {
+    if (displayedTags.length === 0) return html``;
+    return html`
+      <div class="w-chips">
+        ${displayedTags.map((displayedTag) => {
+          return html`
+            <a class="w-chip" href="${displayedTag.href}"
+              >${displayedTag.chip}</a
+            >
+          `;
+        })}
+      </div>
+    `;
+  }
+
   return html`
-    <a href="${url}" class="w-card">
+    <div class="w-card">
       <article class="w-post-card">
-        <div
-          class="w-post-card__cover ${thumbnail &&
-            `w-post-card__cover--with-image`}"
-        >
-          ${thumbnail && renderThumbnail(url, thumbnail, alt)}
-          <h2
-            class="${thumbnail
-              ? `w-post-card__headline--with-image`
-              : `w-post-card__headline`}"
+        <a class="w-card--link" href="${url}">
```

Comment:
This one is tricky. I see that you had to move the anchor to make the tags clickable, but moving the anchor here makes it so you can't click the text of the card and it breaks the focus styles when you tab through the page.

Would it be possible to move the tags so they are a sibling of `w-post-card__desc`. Then have `w-card__link` wrap `w-post-card__cover` and `w-post-card__desc`.

Also note it should be two `w-post-card__link` (two underscores) because it's a BEM element, not a modifier.

Reply:
So, I was actually happy with the fact the entire card wouldn't be clickable here because of #1967 and also GoogleChrome/lighthouse#/10122

Reply:
Those two issues make sense, but I'm concerned that there would be a decent number of users who would find it confusing because the expectation set by many sites is that the entire card is clickable.

Maybe we can look at some example sites to see what they do. I noticed https://www.washingtonpost.com/ has cards where the text is an anchor but the space around it is not, so you can click the text and drag to select it. 
![image](https://user-images.githubusercontent.com/1066253/73317043-a040ad00-41e9-11ea-9b4e-67049a572cf9.png)

If we wrap the image, headline, and desc in anchors then that may leave enough gaps in the card's padding to let folks go in click-drag to select?